src/traefik_route_observer.py

```diff
@@ -86,7 +86,10 @@ def _ingress_config(self) -> dict[str, dict[str, dict[str, typing.Any]]]:
                 "tls": {"passthrough": True},
             }
             services[service_name] = {
-                "loadBalancer": {"servers": [{"address": f"{self.hostname}:{port}"}]}
+                "loadBalancer": {
+                    "servers": [{"address": f"{self.hostname}:{port}"}],
+                    "terminationDelay": 1000,
```

Comment:
The `terminationDelay` inside `tcp.services.*.loadBalancer` is deprecated in favor of the `terminationDelay` inside `tcp.serversTransports.*`. However, when I patched the Wazuh charm in production using the `tcp.serversTransports` configuration, the Traefik charm didn’t recognize it, as it's using an older version of traefik, which is why I used the deprecated one.  
For a permanent fix, I think we should use the new `tcp.serversTransports` configuration and upgrade the Traefik charm instead of relying on the deprecated configuration.

https://doc.traefik.io/traefik/routing/services/#termination-delay
https://doc.traefik.io/traefik/routing/services/#terminationdelay

Reply:
The thing is that we don't have control over the version of the workload that's running. The current charm is already the latest on edge :/